asf_core_data/analysis/data_loading.ipynb

```diff
@@ -0,0 +1,1979 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "2ebe8bd6",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The autoreload extension is already loaded. To reload it, use:\n",
+      "  %reload_ext autoreload\n"
+     ]
+    }
+   ],
+   "source": [
+    "%load_ext autoreload\n",
+    "%autoreload 2\n",
+    "\n",
+    "import os\n",
+    "\n",
+    "from asf_core_data import load_preprocessed_epc_data, PROJECT_DIR\n",
+    "from asf_core_data.config import base_config\n",
+    "\n",
+    "from asf_core_data.getters import data_getters\n",
+    "from asf_core_data.getters.epc import epc_data, data_batches\n",
+    "from asf_core_data.getters.supplementary_data.deprivation import imd_data\n",
+    "from asf_core_data.getters.supplementary_data.geospatial import coordinates\n",
+    "\n",
+    "from asf_core_data.pipeline.preprocessing import preprocess_epc_data\n",
+    "from asf_core_data.pipeline.preprocessing import feature_engineering, data_cleaning\n",
+    "from asf_core_data.pipeline.data_joining import merge_install_dates\n",
+    "from asf_core_data.utils.visualisation import easy_plotting\n",
+    "\n",
+    "from asf_core_data.utils.visualisation import easy_plotting"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "abc1c05c",
+   "metadata": {},
+   "source": [
+    "## Local Data Dir\n",
+    "\n",
+    "While all the data can be loaded from S3 directly, some data takes a long time to load. It is easiest to keep a local copy in a designated folder that mirrors the structure of the S3 bucket asf-core-data. \n",
+    "\n",
+    "The setup is easy: create an empty folder of any name in any location you like. The next cell will even create this folder for you if it doesn't already exist.\n",
+    "\n",
+    "The keyword argument `data_path` that is passed to many loading functions defines whether the data is loaded from S3 (`\"S3\"`) or from the local directory (`LOCAL_DATA_DIR`). The advantage of loading from S3 is that you data is always up-to-date. The disadvantage is that reading certain data directly from S3 can take a long time, which would need to be repeated everytime the code is executed. \n",
+    "\n",
+    "We recommend using direct loading from S3 for smaller datasets, such as MCS and supplementary data or smaller subsets of the EPC data, but keep a copy of the large EPC files in a local directory. Functions such as `data_batches.check_for_newest_batch()` will notify you if your local batch is not up-to-date. \n",
+    "\n",
+    "**Note**: the keyword `data_path` will be renamed to `data_source` before the next merge.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "bb07b5ca",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "LOCAL_DATA_DIR = '/path/to/dir'\n",
```

Comment:
doesn't work with `LOCAL_DATA_DIR = "/inputs/"`, it yields this error:
<img width="758" alt="image" src="https://user-images.githubusercontent.com/70645959/223378506-5eb3989f-6490-44b7-b833-ccc5cc6762ba.png">

Reply:
it needs to be the full path to a directory, so might be worth saying that in the text before. :)

Reply:
Well, it needs to be a full path or a relative path (in relation to where script is run), for instance `../../inputs`. But that's always the case.